src/lib/forward.ts

```diff
@@ -124,8 +124,8 @@ export class EventForwarder {
       const events: ZWaveNodeEvents[] = [
         "value updated",
         "value added",
-        "value notification",
         "value removed",
+        "value notification",
```

Comment:
This is now not a change?

Reply:
No, with the consolidating the dumpValue this can all be handled